src/material-experimental/mdc-chips/chip-row.ts

```diff
@@ -145,4 +177,21 @@ export class MatChipRow extends MatChip implements AfterContentInit, AfterViewIn
         this._handleInteraction(event);
     }
   }
+
+  protected _onEditStart() {
+    // Defer initializing the input so it has time to be added to the DOM.
+    setTimeout(() => {
+      this.editInput.initialize(this.value);
+    });
+  }
+
+  protected _onEditFinish() {
+    // If the edit input is still focused or focus was returned to the body after it was destroyed,
+    // return focus to the chip contents.
+    if (document.activeElement === this.editInput.getNativeElement() ||
```

Comment:
inject in the constructor for the sake of SSR

Reply:
Done